core/src/main/resources/lib/form/radio.jelly

```diff
@@ -43,9 +43,14 @@ THE SOFTWARE.
       toggles the radio.
     </st:attribute>
   </st:documentation>
-  <input type="radio" name="${attrs.name}" onclick="${attrs.onclick}" id="${attrs.id}" value="${attrs.value}"
-        checked="${attrs.checked?'true':null}" disabled="${readOnlyMode?'true':null}" />
-  <j:if test="${attrs.title!=null}">
-    <label class="attach-previous">${attrs.title}</label>
-  </j:if>
+  <j:set var="itemId" value="${attrs.id ?: h.generateItemId()}" />
```

Comment:
This seem to cause https://issues.jenkins.io/browse/JENKINS-70139.

Is this value (or this templated) cached when it is used in a repeatable?

Reply:
likely it's cloned and it needs updating